ska_conda/pkg_defs/ska3-core/meta.yaml

```diff
@@ -35,11 +32,12 @@ requirements:
    - freetype ==2.5.5
    - git ==2.11.1
    - glib ==2.50.2
-   - gst-plugins-base ==1.8.0
-   - gstreamer ==1.8.0
+   - gst-plugins-base ==1.8.0 [linux]
+   - gstreamer ==1.8.0 [linux]
    - h5py ==2.7.0
    - hdf5 ==1.8.17
-   - html5lib ==0.9999999
+   - html5lib ==0.9999999 [linux]
```

Comment:
The `html5lib` folks need to commit to getting 1.0 out!!

Reply:
Hah :smile: